test/utils/utils.ts

```diff
@@ -0,0 +1,11 @@
+import * as fs from "fs";
+import * as child from "child_process";
+import * as path from "path";
+
+export function getWasmInstance(sourcePath: string, outputFile?:string): WebAssembly.Instance {
```

Comment:
Being synchronous this can take a very long time, certainly ok in the short term but maybe this should be named getWasmInstanceSync so it makes it more obvious?